helios-client/src/main/java/com/spotify/helios/client/HeliosClient.java

```diff
@@ -563,22 +564,28 @@ public Builder setClientCertificatePath(final ClientCertificatePath clientCertif
       return this;
     }
 
+    public Builder setExecutorService(
+        final ListeningScheduledExecutorService executorService) {
```

Comment:
this signature could probably just be `ScheduledExecutorService` and we wrap it in `listeningDecorator()`, I'm not sure that callers would care or have a LSES beforehand